basic.graphql

```diff
@@ -0,0 +1,15 @@
+schema
+  @core(feature: "https://lib.apollo.dev/core/v0.1")
+  @core(feature: "https://specs.example.com/example/v1.0")
+{
+  query: Query
+}
+
+type Query {
+  field: Int @example
+}
+
+directive @example on FIELD
```

Comment:
EXAMPLE: I think this should be FIELD_DEFINITION. (Search the whole PR for `on FIELD`; I think all cases should be `FIELD_DEFINITION` and I did not report them all.)

Reply:
Fixed

Reply:
It looks like you caught all the instances except this one itself.